src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/GardenNpcPrices.java

```diff
@@ -72,6 +89,24 @@ public void onGardenNpcPrices(ItemTooltipEvent event) {
 			event.toolTip.addAll(prices.get(tooltipCopy));
 		}
 	}
+
+	public Object[] calculateCopper(int amount) {
```

Comment:
Maybe this return type could be replaced with either a specific data class or something like a `net.minecraft.util.Tuple<String, Double>` (or even just a `Map.Entry<String,Double>` directly)